public/app/plugins/datasource/influxdb/datasource.ts

```diff
@@ -178,11 +178,11 @@ export default class InfluxDatasource extends DataSourceApi<InfluxQuery, InfluxO
         const expandedQuery = {
           ...query,
           datasource: this.name,
-          measurement: this.templateSrv.replace(query.measurement, null, 'regex'),
+          measurement: this.templateSrv.replace(query.measurement, scopedVars, 'regex'),
         };
```

Comment:
I wasn't able to test influx solution as I can not make devenv influx work for me. if reviewer could test it out, it would be amazing! 🧡In the meantime, I will try to make influx work for me.